docs/Developers/Writing_Linter_Bears.rst

```diff
@@ -215,6 +215,12 @@ understands it. This is possible via the ``severity_map`` parameter of
 ``coalib.results.RESULT_SEVERITY`` contains three different values, ``Info``,
 ``Warning`` and ``Error`` you can use.
 
+.. note::
+
+    ``severity_map`` is case insensitive. For example passing
+    ``{'E': RESULT_SEVERITY.MAJOR}`` or ``{'e': RESULT_SEVERITY.MAJOR}``
+    to the severity_map, makes no difference.
```

Comment:
Use ``inline literals`` here also (severity_map)

Reply:
Also what do you think whether merging this `Note` with line `224` ` Note` will be suitable or not?
IMO it's better to use use one `Note` instead of two consecutive ones.

Reply:
Merging would improve the appreance as well.